content/language/word-list/tribe.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Tribe
+---
+
+**Definition**
+
+*Tribe* is a term that can be associated with colonialism or can be considered cultural appropriation.
```

Comment:
I like this practice of using italics the first time the word appears in the document; this is (I'm sure Larry knows) the common practice in technical materials -- denoting the first usage of a term by italicizing it and also having that place in the document either define the term or make it clear what it means in that context.